sdk/reservations/Azure.ResourceManager.Reservations/src/Generated/RestOperations/AzureReservationAPIRestOperations.cs

```diff
@@ -42,13 +42,15 @@ internal HttpMessage CreateGetCatalogRequest(string subscriptionId, string reser
         {
             var message = _pipeline.CreateMessage();
             var request = message.Request;
+            // SqlDatabases is currently not supported in 2022-03-01 version, using 2020-10-01-preview for it instead
```

Comment:
Unfortunately, we don't allow you to manually change the generated code in the `Generated` folder. Do we have a timeline to support SqlDatabases in `2022-03-01`.

Reply:
Reverted back. We will be supporting in 2022-03-01 version soon